From: NARUSE, Yui <naruse@users.sourceforge.jp>
Date: Wed, 21 Jan 2009 15:40:49 +0000 (+0900)
Subject: NKF.xs must follow nkf.c doesn't have WISH_TRUE AND NO_X0201.
X-Git-Url: http://git.sourceforge.jp/view?p=nkf%2Fnkf.git;a=commitdiff_plain;h=f06661980bb179714f60338e09d48ae66ca15f97

NKF.xs must follow nkf.c doesn't have WISH_TRUE AND NO_X0201.

fixed by obache [sf.jp#14731]
---

diff --git a/NKF.mod/NKF.xs b/NKF.mod/NKF.xs
index 626b634..1de2a7c 100644
--- a/NKF.mod/NKF.xs
+++ b/NKF.mod/NKF.xs
@@ -134,9 +134,6 @@ nkf(...)
     data = SvPV(ST(argc),i_len);
     input_ctr = 0;
 
-    if(x0201_f == WISH_TRUE)
-         x0201_f = ((!iso2022jp_f)? TRUE : NO_X0201);
-
     /* allocate the result buffer */
 
     /* During conversion, stirngs length may grow. This is the unit */
@@ -178,9 +175,6 @@ nkf_continue(...)
     data = SvPV(ST(0),i_len);
     input_ctr = 0;
 
-    if(x0201_f == WISH_TRUE)
-         x0201_f = ((!iso2022jp_f)? TRUE : NO_X0201);
-
     /* allocate the result buffer */
 
     /* During conversion, stirngs length may grow. This is the unit */
